dotnet/content/force-stop-web-app.ps1

```diff
@@ -0,0 +1,70 @@
+
+# Prevent the progress meter from trying to access the console mode
+$ProgressPreference = "SilentlyContinue"
+
+$w3wpProcesses=@(Get-Process w3wp)
+
+foreach ($w3wp in @($w3wpProcesses)) 
+{	
+  $w3wp_id=$w3wp.Id
+  # The variables match exactly, so we need to examine loaded modules
+  $w3wp_modules=$w3wp.Modules
+  
+  Write-Output "Examining ${w3wp_id} as shutdown candidate"
+
+  $wellknown_scm_module_count=0;
+  $minimum_scm_module_count=5;
```

Comment:
I am not a huge fan of checking on the loaded modules, but this was the only reliable way I found to examine from powershell.
The environment variables on StartInfo are exact clones.